pr-data.csv

```diff
@@ -1226,7 +1226,16 @@ https://github.com/apache/dubbo,737f7a7ea67832d7f17517326fb2491d0a086dd7,dubbo-c
 https://github.com/apache/dubbo,737f7a7ea67832d7f17517326fb2491d0a086dd7,dubbo-common,org.apache.dubbo.common.logger.support.FailsafeLoggerTest.testFailSafeForLoggingMethod,OD,,,https://github.com/TestingResearchIllinois/idoft/issues/5
 https://github.com/apache/dubbo,737f7a7ea67832d7f17517326fb2491d0a086dd7,dubbo-common,org.apache.dubbo.common.logger.support.FailsafeLoggerTest.testSuccessLogger,UD,,,
 https://github.com/apache/dubbo,737f7a7ea67832d7f17517326fb2491d0a086dd7,dubbo-serialization/dubbo-serialization-fst,org.apache.dubbo.common.serialize.fst.FstObjectInputTest.testEmptyByteArrayForEmptyInput,OD-Vic,Accepted,https://github.com/apache/dubbo/pull/2815,
-https://github.com/apache/dubbo,737f7a7ea67832d7f17517326fb2491d0a086dd7,dubbo-common,org.apache.dubbo.common.threadlocal.InternalThreadLocalTest.testSize,NIO;OD-Vic,Accepted,https://github.com/apache/dubbo/pull/6936,https://github.com/apache/dubbo/pull/9265 fixed the OD-Vic behavior
```

Comment:
Why remove this previously fixed test?

Reply:
This was a new polluter which victimized this test

Reply:
I don't understand. If the test was listed here, it was probably a victim itself. A test could be both (1) a victim of some other polluter and (2) a polluter for some other victim. Someone may have previously fixed it as victim, and you may be now finding it to be a polluter. But it's not a reason to remove prior work on fixing it as a victim. Maybe I'm misunderstanding.

Reply:
I am sorry. I shouldn't have removed it. I'll add more information in the notes. This test was initially self-polluted, and [the fix](https://github.com/apache/dubbo/pull/6936) only addressed this specific test. However, the fix I made has a different polluter and this is one of the victims.

Reply:
Professor @darko-marinov , I have updated the notes to state the history. Could you please let me know if I have to update it?

Reply:
Sorry I still don't understand. The old text was
`NIO;OD-Vic,Accepted,https://github.com/apache/dubbo/pull/6936,https://github.com/apache/dubbo/pull/9265 fixed the OD-Vic behavior`
I assume it means 6936 fixed `NIO` and 9265 fixed `OD-Vic`

The new text is:
`NIO;OD-Vic,Accepted,https://github.com/apache/dubbo/pull/13361,https://github.com/apache/dubbo/pull/9265 fixed the OD-Vic behavior but was victimized by another polluter;https://github.com/apache/dubbo/pull/6936;https://github.com/apache/dubbo/issues/13359`
Should the `Notes` be something like `https://github.com/apache/dubbo/pull/6936 fixed NIO;https://github.com/apache/dubbo/pull/9265 fixed OD-Vic but another polluter was added later;https://github.com/apache/dubbo/issues/13359 reported the new polluter;https://github.com/apache/dubbo/issues/13361 fixed OD-Vic for the new polluter`

Reply:
1st Fix - https://github.com/apache/dubbo/pull/6936 Fixed `NIO` (self polluted) made on Nov 22, 2020
2nd Fix - https://github.com/apache/dubbo/pull/9265 Fixed `OD-Vic` (3 polluters - `testSetAndGet`, `testRemove`, `testRemoveAll`)  made on Nov 13, 2022
3rd Fix *This PR* - https://github.com/apache/dubbo/pull/13361 Fixed `OD-Vic` (1 polluter - `testConstructionWithIndex `) made on Nov 14, 2023

Reply:
Professor @darko-marinov , as discussed in https://campuswire.com/c/G7A0E96FD/feed/662, I have updated the notes.